### DISCUSSION TITLE
I have a problem with sqlcipher.bundle (Multiple commands produce) in my flutter iOS app

### DISCUSSION THREAD

**Discussion by bername** [tekartik/sqflite#1130] [Q&A]
So I have a flutter app and when I debug it on my iPhone it works perfectly but when I try to build an ipa file it shows me this error:

Which indicates that there are multiple commands which are trying to produce the SQLCipher.bundle folder in the same place with the same name. and those two which are attempting that are FMDB and SQLCipher. They are both generated because I'm using sqflite_sqlcipher: ^3.1.0+1 in my flutter app.

Showing Recent Issues Multiple commands produce '/Users/user1/Library/Developer/Xcode/DerivedData/Runner-bgmcokivomqkibdepnikxaoitzbu/Build/Intermediates.noindex/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/SQLCipher.bundle'

Showing Recent Issues Target 'FMDB-SQLCipher' (project 'Pods') has create directory command with output '/Users/user1/Library/Developer/Xcode/DerivedData/Runner-bgmcokivomqkibdepnikxaoitzbu/Build/Intermediates.noindex/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/SQLCipher.bundle'

Showing Recent Issues Target 'SQLCipher-SQLCipher' (project 'Pods') has create directory command with output '/Users/user1/Library/Developer/Xcode/DerivedData/Runner-bgmcokivomqkibdepnikxaoitzbu/Build/Intermediates.noindex/ArchiveIntermediates/Runner/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/SQLCipher.bundle'

and I have tried everything from flutter clean, pub get and upgrade to pod deintegrate and pod install any many more solutions like editing the podfile but unfortunately nothing worked.

Because the both FMDB and SQLCipher are automatically generated when pod install is committed then any changes in podfile.lock would be hopeless.

if anyone has a solution please help.

**Comment by alextekartik** [comment]
You should open the discussion in sqflite_cipher instead: https://github.com/davidmartos96/sqflite_sqlcipher
